src/python/test/test_nextflow.py

```diff
@@ -310,6 +310,23 @@ def test_dx_run_nextflow_with_additional_parameters(self):
         # are not printed
         self.assertNotIn("Launching", watched_run_output)
 
+    # @unittest.skipUnless(testutil.TEST_RUN_JOBS,
```

Comment:
can we add test for resume? Maybe we could run a pipeline and then resume the exact sessionID - as it should always be in the project used, right?

Reply:
https://docs.google.com/document/d/1lx9eLRUVteV32Y3ktHnhCVbCUFwkRIBoXdF3i4wZ0-U/edit# I will need to make unit test suite like what I already have done here. So it is just a placeholder. cc @commandlinegirl